CMakeLists.txt

```diff
@@ -818,7 +818,7 @@ string(REPLACE ";" "\n" _find_dependency_calls "${_find_dependency_calls}")
 write_basic_package_version_file(
 	"${CMAKE_CURRENT_BINARY_DIR}/LibtorrentRasterbar/LibtorrentRasterbarConfigVersion.cmake"
 	VERSION ${libtorrent_VERSION}
-	COMPATIBILITY SameMinorVersion
```

Comment:
It's confusing...
Why does CMake provide this feature at all, so that the package can somehow affect its supporting? Shouldn't it be entirely the user's responsibility to determine which versions it supports?

Reply:
¯\\\_(ツ)_/¯, I'm sure some projects make good use of it.

Reply:
does `COMPATIBILITY` mean ABI or API compatibility?
without knowing really what it means, it seems reasonable to document it in some way, like this. Is the problem that a project that supports multiple versions have no way of expressing all the versions it supports when looking for a dependency?

Reply:
@arvidn 
https://github.com/qbittorrent/qBittorrent/issues/13510#issuecomment-709322633

Reply:
@arvidn It's purely a "dumb" version number comparison policy. It's too inflexible when set to anything other than `AnyNewerVersion`, downstream projects should be the ones that what versions they support above a certain minimum (and they have the means to do this).